Arrays/2779MaximumBeautyofanArrayAfterApplyingOperation.py

```
@@ -0,0 +1,62 @@
+                stack.append(nums[i])
+            else:
+                count=max(count,len(stack))
+                temp=[]
+                for num in stack:
+                    if num-nums[i]<=2*k:
+                        temp.append(num)
+                temp.append(nums[i])
+                stack=temp 
+       
+        return max(count,len(stack)) 
+from collections import defaultdict,Counter
```

Flake8 (7.2.0)

[error] 'collections.Counter' imported but unused

(F401)